src/client/js/script.js

```diff
@@ -63,4 +64,601 @@ $(document).ready(function() {
+
+    $('#nav-main .toggle').removeClass('open');
+
+    // remove submenu click handler and CSS class
+    NavMain.mainMenuLinks
+        .addClass('submenu-item')
+        .unbind('click', NavMain.handleSubmenuClick);
+
+    if (NavMain.currentSmallSubmenu) {
+        NavMain.closeSmallSubmenu(NavMain.currentSmallSubmenu);
+    }
+    NavMain.currentSmallSubmenu = null;
+
+    NavMain.smallMenuOpen = false;
+};
+
+NavMain.handleSubmenuClick = function(e)
+{
+    e.preventDefault();
+    var menu = $(this).siblings('.submenu');
+    NavMain.openSmallSubmenu(menu);
+};
+
+NavMain.openSmallSubmenu = function(menu)
+{
+    // close previous menu
+    if ( NavMain.currentSmallSubmenu
+        && NavMain.currentSmallSubmenu.get(0).id !== menu.get(0).id) {
+        NavMain.closeSmallSubmenu(NavMain.currentSmallSubmenu);
+    }
+
+    // skip current menu
+    if ( NavMain.currentSmallSubmenu
+        && NavMain.currentSmallSubmenu.get(0).id === menu.get(0).id) {
+        // still focus first item
+        menu.find('a').get(0).focus();
+        return;
+    }
+
+    menu
+        .stop(true)
+        .css(
+            {
+                'left'         : '80px',
+                'top'          : 'auto',
+                'display'      : 'none',
+                'opacity'      : '1',
+                'height'       : 'auto',
+                'marginTop'    : '-8px',
+                'marginBottom' : '0'
+            }
+        )
+        .slideDown(150)
+        .attr('aria-expanded', 'true');
+
+    // focus first item
+    menu.find('a').get(0).focus();
+
+    NavMain.currentSmallSubmenu = menu;
+};
+
+NavMain.closeSmallSubmenu = function(menu)
+{
+    menu
+        .stop(true)
+        .fadeOut(100, function() {
+        menu
+	    .css(
+		{
+		    'left'         : '',
+		    'top'          : '',
+		    'display'      : '',
+		    'opacity'      : '',
+		    'height'       : '',
+		    'marginTop'    : '',
+		    'marginBottom' : ''
+		}
+	    )
+            .attr('aria-expanded', 'false');
+    });
+};
+
+$(document).ready(NavMain.init);
+
+})();
+
+/*
+nav-main.js
+*/
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(document).ready(function() {
+  if ($('#nav-main').length === 0) {
+    return;
+  }
+
+  var main_menuitems = $('#nav-main [tabindex="0"]');
+  var prev_li, new_li, focused_item;
+
+  $('#nav-main > .has-submenus > li').bind('mouseover focusin', function(event) {
+    new_li = $(this);
+    if (!prev_li || prev_li.attr('id') !== new_li.attr('id')) {
+      // Open the menu
+      new_li.addClass('hover').find('[aria-expanded="false"]').attr('aria-expanded', 'true');
+      if (prev_li) {
+        // Close the last selected menu
+        prev_li.dequeue();
+      }
+    } else {
+      prev_li.clearQueue();
+    }
+  })/*bind('mouseout focusout', function(event) {
```

Comment:
Yeah. There was no point of having a `focusout` event because we want to close it on click.